activerecord/test/cases/migration_test.rb

```diff
@@ -1015,8 +1015,8 @@ def test_copying_migrations_preserving_magic_comments
     assert File.exist?(@migrations_path + "/4_currencies_have_symbols.bukkits.rb")
     assert_equal [@migrations_path + "/4_currencies_have_symbols.bukkits.rb"], copied.map(&:filename)
 
-    expected = "# coding: ISO-8859-15\n# This migration comes from bukkits (originally 1)"
```

Comment:
Is it worth testing it with more than one magic comments at the beginning of the file?
From what I see in `loop` it should support it.